app/javascript/components/batch_update/components/Selected/SelectedTag.js

```diff
@@ -2,7 +2,27 @@ import React from 'react'
 import SelectedComponent from './SelectedComponent'
 
 const SelectedTag = props => {
-  return <SelectedComponent name={props.name} onRemove={props.onRemove} />
+  const toAdd = props.toAdd
+  const toRemove = props.toRemove
+  const styleOverrides = {}
+  let name = props.name
+  if (toAdd) {
+    styleOverrides.color = 'green'
+    name = `+${name}`
+  }
+
+  if (toRemove) {
+    styleOverrides.red = 'red'
+    name = `-${name}`
+  }
+
+  return (
+    <SelectedComponent
+      style={styleOverrides}
+      name={name}
+      onRemove={props.onRemove}
+    />
```

Comment:
I like that you were following an existing UI pattern here — however I think in this case we'd want to borrow from the patterns established in the batch update UI rather than than the query building UI. I have specifics in mind, can show you what I mean.

Reply:
Cool, yeah, there's a lot of streamlining that can happen here honestly. 👍